source/exec.tex

```diff
@@ -5583,6 +5608,13 @@
 in a coroutine with promise type \tcode{U} is expression-equivalent to
 the same expression in a coroutine with promise type \tcode{Promise}.
 \item
+Otherwise, \tcode{\exposid{sender-awaitable}\{\exposid{adapted-expr}, p\}}
+if \tcode{\exposid{has-queryable-await-completion-adaptor}<Expr>}
+and \tcode{\exposid{awaitable-sender}<decltype((\exposid{adapted-expr})), Promise>}
+are both satisfied, where \exposid{adapted-expr} is
+\tcode{get_await_completion_adaptor(get_env(expr))(expr)},
+except that \tcode{expr} is evaluated only once.
```

Comment:
Sprinkling in some soft hyphens (`\-`) into names and optional word breaks (`\brk{}`) after opening parentheses and punctuation should do the trick here.